brave-lists/clean-urls.json

```diff
@@ -158,6 +158,23 @@
             "t"
         ]
     },
+    {
+        "include": [
+            "*://*.x.com/*",
```

Comment:
We should add the `x.com` URLs into the above `twitter.com` block to avoid the repetition.

Reply:
Sure, merged